airflow/www/templates/airflow/dag.html

```diff
@@ -54,40 +54,24 @@ <h4 class="pull-right">
               Back to {{ dag.parent_dag.dag_id }}</a>
           </li>
       {% endif %}
+      <li><a href="{{ url_for("airflow.trigger_dag", dag_id=dag.dag_id, root=root) }}">
+          <span class="glyphicon glyphicon-play-circle" aria-hidden="true"></span>
+        Trigger Dag
+      </a></li>
       <li><a href="{{ url_for("airflow.graph", dag_id=dag.dag_id, root=root) }}">
           <span class="glyphicon glyphicon-certificate" aria-hidden="true"></span>
         Graph View</a></li>
       <li><a href="{{ url_for("airflow.tree", dag_id=dag.dag_id, num_runs=25, root=root) }}">
           <span class="glyphicon glyphicon-tree-deciduous" aria-hidden="true"></span>
         Tree View
       </a></li>
-      <li><a href="{{ url_for("airflow.duration", dag_id=dag.dag_id, days=30, root=root) }}">
-          <span class="glyphicon glyphicon-stats" aria-hidden="true"></span>
-          Task Duration
-      </a></li>
-      <li><a href="{{ url_for("airflow.tries", dag_id=dag.dag_id, days=30, root=root) }}">
```

Comment:
Task Duration and Task Tries may be useful for debugging some DAGs

Reply:
I think Task Duration can give interesting information, can we just leave it in there? or is it too much for Steven? :D